sea-orm-cli/Cargo.toml

```
@@ -46,11 +52,30 @@ regex = "1"
 smol = "1.2.5"
 
 [features]
-default = [ "codegen", "runtime-async-std-native-tls" ]
-codegen = [ "sea-schema/sqlx-all", "sea-orm-codegen" ]
-runtime-actix-native-tls = [ "sqlx/runtime-actix-native-tls", "sea-schema/runtime-actix-native-tls" ]
-runtime-async-std-native-tls = [ "sqlx/runtime-async-std-native-tls", "sea-schema/runtime-async-std-native-tls" ]
-runtime-tokio-native-tls = [ "sqlx/runtime-tokio-native-tls", "sea-schema/runtime-tokio-native-tls" ]
-runtime-actix-rustls = [ "sqlx/runtime-actix-rustls", "sea-schema/runtime-actix-rustls" ]
-runtime-async-std-rustls = [ "sqlx/runtime-async-std-rustls", "sea-schema/runtime-async-std-rustls" ]
-runtime-tokio-rustls = [ "sqlx/runtime-tokio-rustls", "sea-schema/runtime-tokio-rustls" ]
+default = ["codegen", "runtime-async-std-native-tls"]
+async_std = ["async-std"]
```

Is there any reason to create a `async_std` feature? I'd prefer it to be
```suggestion
default = ["async-std", "dotenv", "codegen", "runtime-async-std-native-tls"]
```

---

Yes because for my self i use tokio as async runtime and i won't compile async_std